audio_api/services/tts_service.py

```
@@ -218,29 +350,68 @@ def _get_style_instructions(self, request: AudioRequest) -> str:
+
+            # Basic WAV file validation
+            async with aiofiles.open(file_path, "rb") as f:
+                header = await f.read(12)
+                if not header.startswith(b"RIFF") or b"WAVE" not in header:
+                    raise TTSError("Generated file is not a valid WAV file")
+
+            logger.info(f"Audio file validated: {file_path} ({file_size:,} bytes)")
+
+        except Exception as e:
+            logger.error(f"Audio file validation failed: {str(e)}")
+            raise TTSError(f"Audio file validation failed: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)